pages/common/timew.md

```diff
@@ -0,0 +1,19 @@
+# timew
+
+> Time tracker.
+
+- Start a new stopwatch:
+
+`timew {{item_to_time}} start`
```

Comment:
What is "item_to_time" here ? The man [page](https://taskwarrior.org/docs/timewarrior/syntax.html) doesn't mention anything before the 'start' option.